### DISCUSSION TITLE
[Feature]:  Add skip and play previous buttons on now playing IOS

### DISCUSSION THREAD

**Discussion by nohwolde** [TheWidlarzGroup/react-native-video#4988] [Ideas]
### What problem are you trying to solve?

I would like the ability to modify the lock screen controls for iOS to be able to remove the seek forward 15 seconds and seek backward 15 seconds, and add the ability to use a skip forward to the next video or skip backward to the previous song.

Why it is needed: Because it is cool!

### Proposed solution

Add skip and play previous buttons on the iOS now playing / lock screen controls. Provide the ability to customize the lock screen controls on iOS, removing the seek forward 15 seconds and seek backward 15 seconds controls and adding skip-to-next and skip-to-previous track controls.

### Platforms

iOS

### Additional context

I'm happy to implement this in a PR but which files would I need to modify for this update?

**Possible implementation:** Technical POV how to do it

**Code sample:** Code sample

[a comment by an automated account is not shown]

**Comment by TWG-AI** [comment]
Thanks for the request! The iOS Now Playing / lock screen currently exposes skip-forward and skip-backward controls. Adding next/previous track buttons would require wiring up `MPRemoteCommandCenter`'s `nextTrackCommand` and `previousTrackCommand` handlers — we've noted this for the team to consider.
